db/pre-re/item_group_db.yml

```diff
@@ -45,12 +46,13 @@
 
 Header:
   Type: ITEM_GROUP_DB
-  Version: 3
+  Version: 4
 
 Body:
   - Group: ACCESORY
     SubGroups:
-      - SubGroup: 1
+      - SubGroup: 6
+        Draws: NoReplacement
```

Comment:
All pre-re groups should use true random so it should be "Replacement" here, right? Maybe I'm still misunderstanding the definition of these fields, but to me calling the algorithm that always have the same chances "NoReplacement" and calling the algorithm that actually removes the items without replacing them "Replacement" seems like the opposite of what your intent was.